src/toil/test/src/importExportFileTest.py

```diff
@@ -14,64 +14,85 @@
 
 from __future__ import absolute_import
 
-import uuid
-
+import random
 import shutil
-
-import os
 from toil.common import Toil
 from toil.job import Job
+from toil.leader import FailedJobsException
 from toil.test import ToilTest
 import tempfile
 
 
 class ImportExportFileTest(ToilTest):
     def setUp(self):
-        self.srcFile = "%s/%s" % (tempfile.mkdtemp(), uuid.uuid4())
+        self._tempBaseDir = tempfile.mkdtemp()
```

Comment:
There is a method for creating a temporary directory in ToilTest. Use that and it will do cleanup for you.